packages/datacommons-mcp/datacommons_mcp/server.py

```diff
@@ -210,22 +209,26 @@ async def get_available_variables(
           "status": "SUCCESS",
           "data": {
             "place_dcid": str,
-            "category_variable_ids": list[str],
+            "variable_dcids": list[str],
             "id_name_mappings": dict
           }
         }
 
         In your response, use the id_name_mappings to convert the variable and place dcids to human-readable names.
 
-        You can use the category_variable_ids to get the variables in the requested category (or for "statistics" by default).
+        You can use the variable_dcids to get the variables in the requested category (or for "statistics" by default) or those that match the indicator_desc.
 
-        If the user asks to see the data for this category and there are a high number of variables, pick those most pertinent to the user's query and context.
+        If the user asks to see the data for a category and there are a high number of variables, pick those most pertinent to the user's query and context.
```

Comment:
We might want to get rid of the instructions regarding how to surface this info to a user. That seems like it should be in an agent prompt, not necessarily in the tool description. This could be addressed in a follow up though!